zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -29,6 +29,9 @@ zipkin:
     lookback: ${QUERY_LOOKBACK:86400000}
     # The Cache-Control max-age (seconds) for /api/v1/services and /api/v1/spans
     names-max-age: 300
+    # CORS allowed-origins.
+    allowed-origins: "*"
```

Comment:
I don't think that's considered valid yaml(?) IIRC Spring Boot doesn't like it. Yamllint doesn't like it either...